src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/NetHttpBindingElement.cs

```diff
@@ -65,6 +65,10 @@ public override Binding CreateBinding()
                 ReaderQuotas = ReaderQuotas.Clone(),
             };
 
+            binding.MessageEncoding = MessageEncoding;
+            //WebSocketSettings.ApplyConfiguration(netHttpBinding.WebSocketSettings);
+            // this.ReliableSession.ApplyConfiguration(netHttpBinding.ReliableSession);
```

Comment:
Can you open a general purpose issue for adding more capabilities to ConfigurationManager with a list with check boxes in the main comment (added by adding square brackets at start of line \[\]) and add the currently known missing features. The one's I can think of off the top of my head are behaviors, base addresses, and now WebSockets for NetHttpBinding. As they get fixed, will mark them as done.

Reply:
https://github.com/CoreWCF/CoreWCF/issues/446 => I will sync offline to modify